scripts/test_runner.sh

```diff
@@ -1,24 +1,28 @@
 #!/bin/bash
 
 main () {
-  testList=$(TF_ACC=1 go test github.com/terraform-providers/terraform-provider-vsphere/vsphere | grep "\-\-\- FAIL" | awk '{ print $3 }')
-  count=$(echo $testList | wc -w)
+  eCode=0
+  testList=$(TF_ACC=1 go test github.com/terraform-providers/terraform-provider-vsphere/vsphere | grep "\-\-\- FAIL" | awk '{ print $3 }' | grep TestAcc$1_)
   for testName in $testList; do 
-    runTest $testName
+    runTest $testName  || eCode=1
   done
+  exit $eCode
 }
 
 runTest () {
+  eCode=0
   for try in {1..2}; do
     res=$(TF_ACC=1 go test github.com/terraform-providers/terraform-provider-vsphere/vsphere -v -count=1 -run="$1\$" -timeout 240m)
```

Comment:
Where does this pick up the required env vars for each test?

Would it make sense to run the test like this:
```
 TESTARGS='-run "^(TestAccResourceVSphereVNic_)"' make testacc
```
 in order to take advantage of `tf-vsphere-devrc.mk` ?

Reply:
For this script, I actually prefer using `go test` instead of `make testacc` because if the easier and less verbose output. On CircleCI (or TeamCity), all the env vars are included in the project config. I'm open to feedback on format though. I see using `make testacc` would be nice for running the tests locally, but hopefully we won't nee to be running the full suite locally very often.